build.sbt

```diff
@@ -0,0 +1,19 @@
+import org.jetbrains.sbtidea.Keys.*
+
+lazy val inspections =
+  project.in(file("."))
+    .enablePlugins(SbtIdeaPlugin)
+    .settings(
+      version := "1.0.0",
+      scalaVersion := "3.3.0",
+      ThisBuild / intellijPluginName := "Aalto Scala inspections",
+      ThisBuild / intellijBuild      := "231.9011.34",
+      ThisBuild / intellijPlatform   := IntelliJPlatform.IdeaCommunity,
+      Global    / intellijAttachSources := true,
+      Compile / javacOptions ++= "--release" :: "17" :: Nil,
+      intellijPlugins += "com.intellij.properties".toPlugin ,
+      intellijPlugins += "org.intellij.scala".toPlugin,
+      Compile / unmanagedResourceDirectories += baseDirectory.value / "resources",
```

Comment:
If the resources dir is under the src dir you might not need unmanaged resources